src/libraries/System.Data.Odbc/src/System.Data.Odbc.csproj

```diff
@@ -10,13 +10,14 @@
   <PropertyGroup>
     <IsPartialFacadeAssembly Condition="$(TargetFramework.StartsWith('net4'))">true</IsPartialFacadeAssembly>
     <!-- https://github.com/dotnet/arcade/issues/5717 -->
-    <NoWarn Condition="($([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', '$(NetCoreAppCurrent)'))) and '$(TargetsAnyOS)' == 'true'">$(NoWarn);SA1121</NoWarn>
-    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetFramework)' == 'netstandard2.0' or '$(TargetsAnyOS)' == 'true'">SR.Odbc_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
-  </PropertyGroup>
-  <ItemGroup Condition="$(TargetFramework.StartsWith('$(NetCoreAppCurrent)')) or $(TargetFramework.StartsWith('netcoreapp2.0'))" >
+    <NoWarn Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">$(NoWarn);SA1121</NoWarn>
```

Comment:
i dont think we parse out the rid from tfm for netcoreappCurrent, we will need to keep the check to startsWith

Reply:
Look at the previous condition, this NoWarn should only apply to the rid-less NetCoreAppCurrent.